pongObjects_Methods.js

```diff
@@ -0,0 +1,157 @@
+// get canvas properties 
+const cvs = document.getElementById("PongTable");
+const ctx = cvs.getContext("2d");
+
+// Create User Paddle
+// x, y coordinates are used to start where the paddle will be drawn
+//width, height is the size of paddle itself
+const userPad = {
```

Comment:
similar variables, similar code. cleaner way is to create a function that returns this object.

i.e 
`function createPaddle(x, y) {
    return {
        x: x,
        y: y,
        width: 10,
        height: 100,
        color: 'WHITE',
        score: 0,
    }
}`

and then do

`var userPaddle = createPaddle(0, cvs.height/2 - 100/2)`
`var aiPaddle = createPaddle(cvs.width - userPad.width, cvs.height/2 - 100/2)`